build/edge/Dockerfile

```diff
@@ -17,13 +17,10 @@ github.com/kubeedge/kubeedge/edge/cmd/edgecore
 
 FROM ${RUN_FROM}
 
-LABEL maintainer="zhanghongtong <zhanghongtong@foxmail.com>"
```

Comment:
Please remove the personal maintainer info here, or change it to community info.

Reply:
Done.